docs/tmp-env-setup.md

```diff
@@ -0,0 +1,23 @@
+With the new microservices shipping, the old [environment-setup](environment-setup) document might not work. In the transition, this document may be helpful for developers. Here are some instructions for different scenarios, they are based on [next.config.js](../src/web/next.config.js) code logic. This `next.config.js` file configures which `env` file to use. It reads `.env` in the root first, if there is no `.env` in the root, it reads [env.development](../config/env.development).
+
+- Only want to work on front-end, don't need anything from back-end.
+
+  ```
+  # 1. copy config/env.staging to .env.
+  # If you have old .env, it is good to delete it before copy to prevent overwrite error
+  cp config/env.staging .env
+  # 2. start services
+  npm run services:start
```

Comment:
We don't have to run microservices locally while only developing the frontend.

Reply:
I disagree. What you're saying only applies to developers working on UI changes in the front-end, in which case, the services aren't needed. But if they're making a change that requires a service to run, such as the search service in order to test the search page, then they should run the specific service needed

Reply:
@yuanLeeMidori says this if for`Only want to work on front-end, don't need anything from back-end.`. I don't see why we need to run services locally here.

Reply:
Sorry that I make you confused. Like @HyperTHD said, for most front-end changes, they still need the service running in order to get the posts, for example. In my opinion, it is rare that developers can work with just pure front-end without service running to provide functionalities.

How about, `Only want to work on front-end, and won't make any changes on the back-end.`?

Reply:
I think that's how I understand it. When you `# 1. copy config/env.staging to .env.`, your frontend is already using services running on Vercel. Correct me if I'm wrong but on my machine, I can develop frontend and see posts and banner without opening running docker.

Reply:
@DukeManh Just tried it, you're right. I'll update it!